src/linear_algebra/basis.lean

```diff
@@ -402,6 +437,9 @@ apply linear_equiv.of_bijective (linear_map.cod_restrict (span α (range v)) (fi
   apply mem_range_self l }
 end
 
+/- Given a family of linearly independent vectors, we can represent any vector in their span as
+   a linear combination of these vectors. These are provided by this linear map.
+   It is simply one direction of `linear_independent.total_equiv` -/
```

Comment:
I'm not sure this style of docstring will be very nice to read in VScode popups when hovering on an identifier.

Reply:
Fortunately nobody will have to read it that way since it doesn't start with `/--`...

Reply:
I changed to this:

```
/-- Linear combination representing a vector in the span of linearly independent vectors.

   Given a family of linearly independent vectors, we can represent any vector in their span as
   a linear combination of these vectors. These are provided by this linear map.
   It is simply one direction of `linear_independent.total_equiv` -/
```

Or would you prefer to remove the second paragraph entirely?